src/frontend/state/blogs/reducer.js

```diff
@@ -0,0 +1,19 @@
+import * as actionTypes from '../action-types';
+
+const initialState = {
+  allBlogInfo: [],
+};
+
+const reducer = (state = initialState, action) => {
+  switch (action.type) {
+    case actionTypes.GET_BLOG_INFO:
```

Comment:
Is it appropriate to rename this to `UPDATE_BLOG_INFO` ?

Reply:
Yes I think you're right